pkg/virt-controller/watch/workload-updater/workload-updater_test.go

```diff
@@ -268,8 +265,7 @@ var _ = Describe("Workload Updater", func() {
 				}
 			}
 
-			// wait for informer to catch up since we aren't watching for vmis directly
-			time.Sleep(1 * time.Second)
+			waitForNumberOfInstancesOnVMIInformerCache(controller, 50)
```

Comment:
here I would probably just have a `const desiredNumberOfVMs = 50` and use it both in the for and in the `waitForNumberOfInstancesOnVMIInformerCache` .

Reply:
Done, thank you!